Ginger/Ginger/BusinessFlowPages_New/NewAutomatePage.xaml.cs

```diff
@@ -64,8 +64,9 @@ public partial class NewAutomatePage : Page
         VariabelsListViewPage mBfVariabelsPage;
         BusinessFlowConfigurationsPage mBfConfigurationsPage;
         ActivityPage mActivityPage;
+        MainAddActionsNavigationPage MainNavigationPage;
```

Comment:
MainNavigationPage should be "mMainNavigationPage"

Reply:
Renamed